### PR TITLE
Add/update info about PaginatorComponent deprecation.

### PR DESCRIPTION
**Pull request by ADmad**

### CODE REVIEW

File: en/controllers/pagination.rst [hunk 1]
```diff
@@ -0,0 +1,294 @@
+Pagination
```
Comment (ADmad) [inline en/controllers/pagination.rst:1]: This is basically a copy of `controllers/component/pagination` with references to the component removed.

File: en/appendices/4-4-migration-guide.rst [hunk 2]
```diff
@@ -92,6 +92,13 @@ be removed in 5.0.
 - Use :ref:`controller-viewclasses` instead of defining view class mappings in
   ``RequestHandlerComponent``.
 
+PaginationComponent
+-------------------
+
+The ``PaginationComponent`` is deprecated and will be removed in 5.0.
+Use the ``Controller::$paginate`` property of the ``$settings`` parameter of
```
Comment (markstory) [inline en/appendices/4-4-migration-guide.rst]:
```suggestion
Use the ``Controller::$paginate`` property or the ``$settings`` parameter of
```

File: en/contents.rst [hunk 3]
```diff
@@ -43,7 +43,7 @@ Contents
     core-libraries/internationalization-and-localization
     core-libraries/logging
     core-libraries/form
-    controllers/components/pagination
+    controllers/pagination
```
Comment (markstory) [inline en/contents.rst:46]: You might get warnings that this document isn't in a TOC. If so you can add a hidden toc

```rst

.. toctree::
    :hidden:
    
    controllers/components/pagination
```

Should fix the warnings.

Comment (ADmad) [reply en/contents.rst:46]: The warnings were due to duplicate labels, fixed now.

**Review by Arhell — APPROVED** [review]

**Pull request merged**